Library/Homebrew/compat/md5.rb

```diff
@@ -1,26 +1,38 @@
 class Formula
   def self.md5(val)
+    odie <<-EOS.undent
+      Using MD5 has been dropped for security reasons.
+      Please switch to SHA-256.
+    EOS
```

Comment:
I don't think we should do this, which has been discussed before https://github.com/Homebrew/legacy-homebrew/pull/38433#issuecomment-90551522

One example is, if user's have a formula using `md5`, with this change. `brew upgrade` or any command which will load this formula will just exit unexpectedly.